nix/deps/nodejs-patched/default.nix

```
@@ -66,6 +67,12 @@ stdenv.mkDerivation {
   patchJavaPhase = ''
     ${nodejs}/bin/node ./node_modules/jetifier/bin/jetify
   '';
+  # Patch node modules using `patch-package`
```

This comment is inaccurate.

Also, you should have the comment explain WHY something is being done, not WHAT, we can see WHAT is done.
A correct comment would be something like:
```suggestion
  # FIXME: Remove this once release newer than 1.19.0 is used which includes:
  # https://github.com/facebook/yoga/commit/f174de70
```